jmh-core/src/main/java/org/openjdk/jmh/generators/core/BenchmarkGenerator.java

```diff
@@ -318,7 +318,7 @@ private void validateBenchmark(ClassInfo clazz, Collection<MethodInfo> methods)
             if (m.getAnnotation(Group.class) != null && m.getAnnotation(Threads.class) != null) {
                 throw new GenerationException("@" + Threads.class.getSimpleName() + " annotation is placed within " +
                         "the benchmark method with @" + Group.class.getSimpleName() + " annotation. " +
-                        "This has ambiguous behavioral effect, and prohibited. " +
+                        "This has ambiguous behavioral effect and is prohibited. " +
```

Comment:
I would change this to "this is prohibited because it is ambiguous."

Reply:
the correct article on line 320 would be "**a** benchmark method with @"

Reply:
Thank you. I made these changes.